bindings/python/test/test_arrow.py

```diff
@@ -87,6 +87,16 @@ def test_find_projection(self):
         self.assertEqual(find_cmd.command_name, 'find')
         self.assertEqual(find_cmd.command['projection'], projection)
 
+    def test_find_with_session(self):
+        with self.client.start_session() as session:
+            last_use = session._server_session.last_use
```

Comment:
`session._server_session` is private so we shouldn't use it unless we have to. Instead you can use the `session.operation_time` property:
```
self.assertIsNone(session.operation_time)
...
self.assertIsNotNone(session.operation_time)
```

Reply:
Updated